src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1611,7 +1611,7 @@ static uint8_t addHue(uint8_t hue1, uint8_t hue2)
 
     if (hue16 > MAX_HUE_VALUE)
     {
-        hue16 = static_cast<uint16_t>(hue16 - MAX_HUE_VALUE);
+        hue16 = static_cast<uint16_t>(hue16 - UINT8_MAX);
```

Comment:
Shouldn't this be `hue16 - MAX_HUE_VALUE - 1`, so we don't assume `MAX_HUE_VALUE == UINT8_MAX - 1`?